### ISSUE TITLE
Confirmation box freezes visual interface on Linux

### ISSUE THREAD

**Issue by tiagoornelas** [mongodb-js/vscode#348]
#### Current Behavior

Maybe hard to reproduce this bug, but while using the extension and running a command, the confirmation box freezes VSCode and OS, clicking is no longer possible in any aplication and I was forced to reboot the machine. After disabling the confirmation box, everything works as expected.

#### Environment

OS: POP_OS 21.04

**Comment by Anemy** [comment]
@tiagoornelas Can you try disabling GPU acceleration in VSCode and see if that solves your issue?
https://code.visualstudio.com/updates/v1_40#_disable-gpu-acceleration

Just to clarify, which command are you running? Do you mean the confirmation box about running a playground query?

Thanks for reporting the issue, hopefully we can solve it soon!

**Comment by tiagoornelas** [comment]
Hello, @Anemy ! I was talking about the playground's confirmation box below.

I tried disabling the GPU acceleration and enabling the confirmation box and it works fine. But I also changed the OS in the meantime and now my machine runs on Linux Mint, which made me try without the GPU acceleration and it worked as well.

There were no other confirmation boxes that caused the same problem as far I had experienced on that OS.

So, I'm sorry I cannot help you investigate further about this issue, however, I am assuming the previous OS was responsible somehow.

![image](https://user-images.githubusercontent.com/78763308/140504637-a1eaa377-3a90-49fb-9e98-c8c0d069df4e.png)
